app/models/rentals_item.rb

```diff
@@ -1,11 +1,13 @@
 # frozen_string_literal: true
+
 class RentalsItem < ActiveRecord::Base
   has_paper_trail
   # use inverse of so we can create this with rental blank at first
   belongs_to :rental, inverse_of: :rentals_items
-  belongs_to :item
+  belongs_to :item, optional: true
+  # TODO: remove item_type from rentals_item.
   belongs_to :item_type
-  validates :rental, :item_type, presence: true # we dont validate item, that will be assigned by the rental
+  # we dont validate item, that will be assigned by the rental
```

Comment:
Don't need comment if we removed item_type validation

Reply:
Also even if we remove item_type, shouldn't rental still be validated?

Reply:
All I did was not remove a comment. I don't know what they meant but I decided to tackle whatever they were talking about later, in which case the comment will be helpful.

Reply:
Still leaves the question of removing the validation for rental, just want to double check that it was intended, as I only know you were removing item_type validations.

Reply:
`belongs_to` validates presence by default. So, I wasn't removing anything but redundancy.